configs/test-utils/src/ThrowError.tsx

```diff
@@ -24,7 +34,7 @@ export const ThrowNull = ({ after, children }: ThrowNullProps) => {
   if (isNeedError) {
     throw null
   }
-  useSetTimeout(() => setIsNeedError(true), after)
+  useTimeout(() => setIsNeedError(true), after)
   return <>{children}</>
```

Comment:
is this need for wrapping Fragment component? too!

Reply:
https://github.com/suspensive/react/pull/264#discussion_r1373686578